test/e2e/resource_quota.go

```diff
@@ -41,6 +41,75 @@ const (
 
 var classGold string = "gold"
 
+var _ = framework.KubeDescribe("ResourceQuota [Feature:LocalStorageCapacityIsolation]", func() {
+	f := framework.NewDefaultFramework("resourcequota")
+
+	BeforeEach(func() {
+		// only run the tests when LocalStorageCapacityIsolation feature is enabled
+		framework.SkipUnlessLocalEphemeralStorageEnabled()
+	})
+
+	It("should create a ResourceQuota and capture the life of a pod.", func() {
```

Comment:
Other than ~6 lines this `It` block is a duplicate of the same test case below. Is there a way to consolidate these 2 It blocks? Maybe a table-driven tests or helper funcs?